parse_data.py

```diff
@@ -2,6 +2,9 @@
 import json
```

Comment:
This file could also use a brief summary of the purpose. (To convert input files to numerical JSON data for hardware?)

Reply:
Yes - this file is still a work in progress, but I have added a brief description of what it (will) do.